pkg/driver/efs_watch_dog.go

```diff
@@ -214,7 +219,9 @@ func (w *execWatchdog) updateConfig(efsClientSource string) error {
 		return fmt.Errorf("cannot create config file %s for efs-utils. Error: %v", w.efsUtilsCfgPath, err)
 	}
 	defer f.Close()
-	efsCfg := efsUtilsConfig{EfsClientSource: efsClientSource}
+	// used on Fargate, IMDS queries suffice otherwise
+	region := os.Getenv("AWS_REGION")
```

Comment:
@kbasv i wonder if we could add support for querying fargate task metadata to efs-utils? https://github.com/aws/efs-utils/blob/c5abeed1a90a20c0ac0ad1a592296f795cd33e7c/src/mount_efs/__init__.py#L293 will create issue later

Reply:
Should be possible since efs-utils queries ECS metadata for credentials - https://github.com/aws/efs-utils/blob/c5abeed1a90a20c0ac0ad1a592296f795cd33e7c/src/mount_efs/__init__.py#L442

Reply:
@anonymint the comment "// used on Fargate, IMDS queries suffice otherwise" is not 100% clear, are you saying it is guaranteed that AWS_REGION will be set if this application is run as a fargate task/pod?

Reply:
@wongma7 ythere is another [PR not merged](https://github.com/kubernetes-sigs/aws-efs-csi-driver/pull/227) regarding this and I would change this to be `$AWS_DEFAULT_REGION` as current setup in Fargate today, will update accordingly and send you package that Fargate uses and guarantee to have it. 

on the side note, IMHOM It would be  cleaner to add in attached efs-utils as you mention, there should be similar way/location to pull this region.